au4/src/projectscene/view/timeline/timelinecontext.cpp

```diff
@@ -558,12 +559,39 @@ double TimelineContext::selectionEndTime() const
     return m_selectionEndTime.raw();
 }
 
+double TimelineContext::selectionStartPosition() const
+{
+    return m_selectionStartPosition;
```

Comment:
I think it's better to do it simply here (not introduce a new variable)
```
return timeToPosition(m_selectionEndTime);
```

And you will only need to send (emit) a notification of the change
one when changing start time
second when changing zoom 

same for `end`